### PR TITLE
Refactor settings to common folder and ignored sensitive settings

### PR DESCRIPTION
**Pull request by nackster12**
Will send local_settings and production_settings separately

### CODE REVIEW

File: colonialsite/colonialsite/settings.py [hunk 1]
```diff
@@ -15,59 +15,109 @@
...
-
-SOPHOMORE_YEAR = '2019'
-JUNIOR_YEAR = '2018'
-SENIOR_YEAR = '2017'
+# TIGERBOOK_LOGIN
+# TIGERBOOK_PW
 
 # Used in coloauth.middleware
-# Default Login is automatically added
+# Default Login is automatically added
```
Comment (aruchames) [inline colonialsite/colonialsite/settings.py]: Did you add this trailing whitespace back in

File: colonialsite/colonialsite/settings.py [hunk 2]
```diff
@@ -15,59 +15,109 @@
...
+# --- URLS ---
+
+LOGIN_URL = '/accounts/login/'
+CAS_SERVER_URL = 'https://fed.princeton.edu/cas/'
+ROOT_URLCONF = 'colonialsite.urls'
+
+
+# --- EMAIL AND LOGGING ---
+
+EMAIL_HOST = 'smtp.gmail.com'
```
Comment (aruchames) [inline colonialsite/colonialsite/settings.py]: Should this be in the dev settings? Are you going to start sending failure logs from dev environments?

Comment (nackster12) [reply colonialsite/colonialsite/settings.py]: Logging is removed from local settings but it's fair that these settings are unnecessary; I'll move them to production_settings only

File: colonialsite/colonialsite/settings.py (hunk 1 above)
Comment (nackster12) [reply colonialsite/colonialsite/settings.py]: Sorry, I copypasta'd from an old version.

**Pull request merged**